docker-compose.yml

```diff
@@ -5,35 +5,24 @@ services:
     container_name: hapi-fhir-jpaserver-start
     restart: on-failure
     ports:
-      - '8080:8080'
+      - "8080:8080"
     environment:
-      datasource.driver: 'com.mysql.jdbc.Driver'
-      datasource.password: 'admin'
-      datasource.url: 'jdbc:mysql://hapi-fhir-mysql/hapi'
-      datasource.username: 'admin'
-      fhir_version: 'DSTU2'
-      hapi.fhir.server_address: 'http://localhost:8080/fhir/'
-      hibernate.dialect: 'org.hibernate.dialect.MySQL5InnoDBDialect'
-      reuse_cached_search_results_millis: '0'
-      spring.config.location: 'classpath:/application-custom.yaml'
-      subscription.resthook.enabled: 'true'
-      subscription.websocket.enabled: 'true'
-      url_pattern: "/fhir/*"
-      enable-web: false
-  hapi-fhir-mysql:
-    image: mysql:latest
-    container_name: hapi-fhir-mysql
-    #https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html
-    command: --lower_case_table_names=1
+      fhir_version: 'R4'
+      hapi.fhir.server_address: 'http://localhost:8080/R4/'
```

Comment:
This should be 'http://localhost:8080/baseR4'. I don't know if the ending `/` is significant, but the upstream repo never uses a trailing `/` for these.